packages/web/src/components/listView.tsx

```diff
@@ -143,6 +142,7 @@ export function ListView<T>({
         const rendered = render(item, index);
         return <div
           key={id?.(item, index) || index}
+          onDoubleClick={() => onAccepted?.(item, index)}
```

Comment:
Why this change?

Reply:
The `ListView`'s `onAccepted` handler was before only working when it was acting in 'select mode'. This means `selectedItem` was there. Now it also works without.